addons/web/controllers/main.py

```diff
@@ -1884,6 +1888,12 @@ class CSVExport(ExportFormat, http.Controller):
     def index(self, data, token):
         return self.base(data, token)
 
+    @http.route('/web/async_export/csv', type='json', auth="user")
+    def index_async(self, data, token):
+        async_ = request.env['ir.async'].with_context(request.context)
```

Comment:
Is that really useful? AFAIK `request.env` is built from `reqyest.context` already.